internal/app/client/conformance_client.go

```diff
@@ -221,23 +221,29 @@ func (i *invoker) clientStream(
 	}, nil
 }
 
-func (i *invoker) bidiStream(
+func (i *invoker) bidiStream( //nolint:nonamedreturns
```

Comment:
IMO, the "nonamedreturns" linter is _never_ valuable. Let's just add it to the exclude list, like so:
https://github.com/bufbuild/httplb/blob/main/.golangci.yml#L48